src/index.tsx

```diff
@@ -452,6 +465,60 @@ export class App extends React.Component<{}, AppState> {
               </>
             )}
 
+            {hasText(elements) && (
+              <>
+                <h5>Font size</h5>
+                <ButtonSelect
+                  options={[
+                    { value: 16, text: "16 px" },
+                    { value: 20, text: "20 px" },
+                    { value: 24, text: "24 px" },
+                    { value: 28, text: "28 px" },
+                    { value: 32, text: "32 px" },
+                    { value: 36, text: "36 px" },
+                    { value: 40, text: "40 px" },
+                    { value: 50, text: "50 px" },
+                    { value: 60, text: "60 px" }
+                  ]}
+                  value={getSelectedAttribute(
+                    elements,
+                    element => isTextElement(element) && +element.font.split("px ")[0]
+                  )}
+                  onChange={value =>
+                    this.changeProperty(element => {
+                      if(isTextElement(element)) {
+                        element.font = `${value}px ${element.font.split("px ")[1]}`;
+                        this.redrawTextBoundingBox(element);
+                      }
+                    })
+                  }
+                />
+                <h5>Font familly</h5>
+                <ButtonSelect 
+                  options={[
+                    {value: "Virgil", text: "Virgil"},
```

Comment:
Could you remove everything but Virgil, Helvetica and Courier, the rest don't seem to provide a ton of value to be different

Reply:
It's done